pyatv/scripts/atvui.py

```diff
@@ -0,0 +1,128 @@
+import argparse
```

Comment:
I think we should deal with the UI in a separate PR. I'm all for including it since it does not add any additional dependencies though. Will  have to review and test it separately.

Reply:
Ok I removed the script